### DISCUSSION TITLE
Coloring icons in Timeline

### DISCUSSION THREAD

**Discussion by kawal2266** [UI5/webcomponents#7759] [Q&A]
I only want to color icon in TimeLine , but design property doesn't work , how can I do that ?

**Comment by Lukas742** [comment]
Hello @kawal2266 

I don't think this is possible as the `part` attribute is not available for the `ui5-icon` element within the shadow root (you can find out more about this [here](https://sap.github.io/ui5-webcomponents-react/?path=/docs/knowledge-base-styling--docs#css-shadow-parts)). However, since this component is a web component developed by the UI5 web component team, I will forward this question to their repo.

**Comment by nnaydenow** [comment]
Hi @kawal2266,

Currently we don't have option to specify the color of the icon in ui5-timeline-item. I created feature request and you can track the status there: https://github.com/SAP/ui5-webcomponents/issues/7761

**Comment by kawal2266** [comment]
Thanks @nnaydenow  & @Lukas742 , looking forward to this feature.

Best Regards ,
Kawal

**Comment by DMihaylova** [comment]
Hi @kawal2266,

Won't changing the variable for icon's color suit your needs?
ui5-timeline-item {
 --sapContent_NonInteractiveIconColor: red;
}
[Sample](https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cblxuPGhlYWQ-XG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgXG4gICAgPHN0eWxlPiAgICAgIFxuICAgICAgdWk1LXRpbWVsaW5lLWl0ZW0ge1xuICAgICAgIC0tc2FwQ29udGVudF9Ob25JbnRlcmFjdGl2ZUljb25Db2xvcjogcmVkO1xuICAgICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICA8bWV0YSBjaGFyc2V0PVwiVVRGLThcIj5cbiAgICA8bWV0YSBuYW1lPVwidmlld3BvcnRcIiBjb250ZW50PVwid2lkdGg9ZGV2aWNlLXdpZHRoLCBpbml0aWFsLXNjYWxlPTEuMFwiPlxuICAgIDx0aXRsZT5TYW1wbGU8L3RpdGxlPlxuPC9oZWFkPlxuXG48Ym9keSBzdHlsZT1cImJhY2tncm91bmQtY29sb3I6IHZhcigtLXNhcEJhY2tncm91bmRDb2xvcik7IHBhZGRpbmc6IDJyZW07XCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS10aW1lbGluZT5cbiAgICAgICAgPHVpNS10aW1lbGluZS1pdGVtIHRpdGxlLXRleHQ9XCJjYWxsZWRcIiBzdWJ0aXRsZS10ZXh0PVwiMjAuMDIuMjAxNyAxMTozMFwiIGljb249XCJwaG9uZVwiIG5hbWU9XCJTdGFuaXNsYXZhIEJhbHRvdmFcIiBuYW1lLWNsaWNrYWJsZT48L3VpNS10aW1lbGluZS1pdGVtPlxuICAgICAgICA8dWk1LXRpbWVsaW5lLWl0ZW0gdGl0bGUtdGV4dD1cImNhbGxlZFwiIHN1YnRpdGxlLXRleHQ9XCIyMC4wMi4yMDE3IDExOjMwXCIgaWNvbj1cInBob25lXCIgbmFtZT1cIlN0YW5pc2xhdmEgQmFsdG92YVwiPjwvdWk1LXRpbWVsaW5lLWl0ZW0-XG5cbiAgICAgICAgPHVpNS10aW1lbGluZS1pdGVtIHRpdGxlLXRleHQ9XCJXZWVrbHkgU3luYyAtIENQIERlc2lnblwiIHN1YnRpdGxlLXRleHQ9XCIyNy4wOC4yMDE3ICgxMTowMCAtIDEyOjAwKVwiIGljb249XCJjYWxlbmRhclwiPlxuICAgICAgICAgICAgPHVpNS1sYWJlbD5NUiBTT0YwMiAyLjQzPC91aTUtbGFiZWw-XG4gICAgICAgIDwvdWk1LXRpbWVsaW5lLWl0ZW0-XG5cbiAgICAgICAgPHVpNS10aW1lbGluZS1pdGVtIHRpdGxlLXRleHQ9XCJWaWRlbyBDb25mZXJlbmNlIENhbGwgLSBVSTVcIiBzdWJ0aXRsZS10ZXh0PVwiMzEuMDEuMjAxOCAoMTI6MDAgLSAxMzowMClcIiBpY29uPVwiY2FsZW5kYXJcIiBuYW1lPVwiU3RhbmlzbGF2YSBCYWx0b3ZhXCI-XG4gICAgICAgICAgICBPbmxpbmUgbWVldGluZ1xuICAgICAgICA8L3VpNS10aW1lbGluZS1pdGVtPlxuICAgIDwvdWk1LXRpbWVsaW5lPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuICAgIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIm1haW4uanNcIj48L3NjcmlwdD5cbjwvYm9keT5cblxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvTGFiZWwuanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvVGltZWxpbmUuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1RpbWVsaW5lSXRlbS5qc1wiO1xuXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9waG9uZS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvY2FsZW5kYXIuanNcIjsifX0)

Regards,
Diana
